proposals/lip-0052.md

```diff
@@ -30,329 +31,345 @@ Therefore, providing a unified module to handle, transfer and modify NFTs is a n
 
 ### Technical Glossary
 
-* **Native chain:** with regards to an NFT, this is the chain where the NFT was created.
-* **Native NFT:** with regards to a chain, all NFTs created on this chain.
-* **Foreign chain:** with regards to an NFT, all chains other than the native chain.
+- **Native chain:** with regards to an NFT, this is the chain where the NFT was created.
+- **Native NFT:** with regards to a chain, all NFTs created on this chain.
+- **Foreign chain:** with regards to an NFT, all chains other than the native chain.
 
 ### NFT Module Store
 
 <img src="lip-0052/nft_module_store.png" width="80%">
 
-*Figure 1: The NFT module store is divided into 5 substores. All NFTs held by users are stored sequentially in the user substore with keys given by the user address and the NFT ID.*
+_Figure 1: The NFT module store is divided into four substores. All NFTs held by users are stored sequentially in the user substore with keys given by the user address and the NFT ID._
 
 #### NFT Store
 
-The NFT store contains entries for all NFTs present on the chain, as well as entries for all native NFTs that have been sent cross-chain. Each entry contains three properties, the owner, the locking module ID and the attributes of the NFT. The owner can either be a 20 bytes user address, or a 4 bytes serialization of a chain ID. In the latter case, the token is a native token that has been sent cross-chain and is escrowed.
+The NFT store contains entries for all NFTs present on the native chain, as well as entries for all native NFTs that have been sent cross-chain to a foreign chain. Each entry contains two properties:
 
-The locking module ID stores the information regarding the locking status of the NFT. If the NFT is unlocked, this property will have the value `NFT_NOT_LOCKED`, whereas if the NFT is locked, this property will store the ID of the locking module.
-
-Lastly, the NFT stores an attribute property which can be used by custom applications to store information about the NFT, or modify interactions with the NFT.
+- The `owner` property can either be a `ADDRESS_LENGTH` bytes long user address or a `CHAIN_ID_LENGTH` bytes long chain ID. In the latter case, the NFT is a native NFT that has been sent cross-chain to a foreign chain and is escrowed.
```

Comment:
I think another option that was proposed was to store the `owner` property in the `attributes` array, under the NFT module. I think this option is fine as well, and at the moment I don't see a specific reason for either one. Maybe it could be simpler to store everything under `attributes` for when we mutate the NFT off-chain?

Reply:
I don't have a specific opinion about this, either. I would leave it as it is for now, and we can touch upon it in the STM.

Reply:
I think it is fine like it is, keeping the `owner` property separate.